examples/create-new-project/main.bal

```diff
@@ -0,0 +1,33 @@
+import ballerina/http;
```

Comment:
Add the license header

Reply:
We expect the example to be a real-world one. You can come up with a simple use case and implement it. There has to be a Readme file per example that explains the scenario. Please refer to Twitter connector examples https://github.com/ballerina-platform/module-ballerinax-twitter/tree/main/examples

Reply:
@Bhashinee Yeah I got your point regarding the example, As I cannot run the examples, I'm not sure what I have done is right (I cant run it because even I import the ballerinax/jira module, it do nit have the path defined yet, so shows an error). So I will make another pull for example and will do some real world examples :)

Reply:
Yeah, Once you update the example, I'll check.